grammars/css.cson

```diff
@@ -516,7 +516,7 @@
         'captures':
           '1':
             'name': 'punctuation.definition.entity.css'
-        'match': '(:+)(after|before|content|first-letter|first-line|host|(-(moz|webkit|ms)-)?selection)\\b'
+        'match': '(::?)(after|before|content|first-letter|first-line|host|(-(moz|webkit|ms)-)?selection)\\b'
```

Comment:
I was gonna the open a PR for this, found you'd already done that. A few things to add:

* `content` and `host` are NOT element selectors; `host` may be a pseudo-class in the future
* every element selector except `after`, `before`, `first-letter` and `first-line` must be preceded by 2 colons—`selection` here should be treated differently
* Gecko (used in Firefox) is the only engine that requires a prefix for `selection`—`webkit|ms` should be removed